### ISSUE TITLE
Extend profile features 

### ISSUE THREAD

**Issue by donutloop** [kennygrant/gohackernews#31]
extend profile features for end users are missing. I would like to show my GitHub, Twitter and LinkedIn account on my own profile page and a picture of me or even do a sync with my GitHub account information. I think it has a positive effect on socializing of an end user.

**Comment by kennygrant** [comment]
I agree this would be nice. Admins can do this, and there is a free text field to add details like that. 

It'd also be nice to allow login via github or twitter so that the user doesn't have to enter info. 

Any other ideas please do post them as issues, thanks for participating on the site.
